benefits/friday_drinks.md

```diff
@@ -0,0 +1,15 @@
+# Friday Drinks
```

Comment:
Should something be linking to this new file?

Reply:
Great catch kind sir!